pyproject.toml

```diff
@@ -38,6 +38,11 @@ dependencies = [
   "jupyterlab==4.0.4",
   "matplotlib==3.7.2",
   "multiprocess==0.70.14",
+  # Matt's additions
+  "pandas==0.24.2",
```

Comment:
Should be pandas>=2.1.0.  Pint is very shy about asking for modern versions of things, but when uncertainties come on line, it's going to need at least this version to work properly.

Reply:
Ah, I think I'd already fixed this in my tree but it didn't make it into this pull request. I'll edit it here...